src/MetaDataAPI/Providers/DelayVault/DelayVaultProvider.cs

```diff
@@ -0,0 +1,23 @@
+﻿using MetaDataAPI.Models.DynamoDb;
+using MetaDataAPI.Models.Response;
+
+namespace MetaDataAPI.Providers;
+
+public class DelayVaultProvider : Provider
+{
+    public override string ProviderName => nameof(DelayVaultProvider);
+    public override string Description => "Description of DelayVaultProvider here.";
```

Comment:
$"The DelayVaultProvider manages the locking of {LeftAmount} tokens for leaderboard purposes. While tokens are locked, users accumulate leaderboard points. Upon withdrawal, point accumulation ceases, and the NFT transitions to a different provider."